auto-wcag/auto-wcag
# Removes outdated accessibility support notes

Removes outdated accessibility notes that some AT did not pick `title` attribute as an accessible name.

Closes issue(s):

- closes #1930 

Need for Call for Review:
This will require a 1 week Call for Review << small changes affecting a small number of test cases, if in doubt do not use this. >>

---

## Pull Request Etiquette

### **When creating PR:**


### **After creating PR:**


### **When merging a PR:**


## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.
Removes outdated accessibility support notes that some ATs did not pick up title as accessible name
